jboss-kie-dashbuilder/added/launch/jboss-kie-dashbuilder.sh

```diff
@@ -0,0 +1,220 @@
+#!/bin/bash
+
+source "${JBOSS_HOME}/bin/launch/launch-common.sh"
+source "${JBOSS_HOME}/bin/launch/logging.sh"
+source "${JBOSS_HOME}/bin/launch/jboss-kie-wildfly-security.sh"
+
+function prepareEnv() {
+    # please keep these in alphabetical order
+    DASHBUILDER_ALLOW_EXTERNAL_FILE_REGISTER
+    DASHBUILDER_COMP_ENABLE
+    DASHBUILDER_COMPONENT_PARTITION
+    DASHBUILDER_DATASET_PARTITION
+    DASHBUILDER_EXTERNAL_COMP_DIR
+    DASHBUILDER_CONFIG_MAP_PROPS
+    DASHBUILDER_IMPORT_FILE_LOCATION
+    DASHBUILDER_IMPORTS_BASE_DIR
+    DASHBUILDER_MODEL_FILE_REMOVAL
+    DASHBUILDER_MODEL_UPDATE
+    DASHBUILDER_RUNTIME_MULTIPLE_IMPORT
+    DASHBUILDER_UPLOAD_SIZE
+    KIESERVER_DATASETS
+    KIESERVER_SERVER_TEMPLATES
+    unset_kie_security_env
+}
+
+function configureEnv() {
+    configure
+}
+
+function configure() {
+    configure_dashbuilder_auth
+    configure_dashbuilder_allow_external
+    configure_dashbuilder_partitions
+    configure_dashbuilder_file_imports
+    configure_dasbuilder_file_import_properties
+    configure_dashbuilder_external_component
+    configure_dashbuilder_kieserver_dataset
+    configure_dashbuilder_kieserver_server_template
+    # must be the last to be executed.
+    dashbuilder_process_properties_file
+}
+
+function configure_dashbuilder_auth() {
+    # add eap user (see jboss-kie-wildfly-security.sh)
+    add_kie_admin_user
+}
+
+function configure_dashbuilder_allow_external() {
+    local allow_external="false"
+    if [ "${DASHBUILDER_ALLOW_EXTERNAL_FILE_REGISTER^^}" = "TRUE" ]; then
+        allow_external="true"
+    fi
+    JBOSS_KIE_DASHBUILDER_ARGS="${JBOSS_KIE_DASHBUILDER_ARGS} -Ddashbuilder.runtime.allowExternal=${allow_external}"
+}
+
+# Configures dashbuilder.dataset.partition and dashbuilder.components.partition properties
+function configure_dashbuilder_partitions() {
+    local component_partition="false"
+    local dataset_partition="false"
+    if [ "${DASHBUILDER_COMPONENT_PARTITION^^}" = "TRUE" ]; then
+        component_partition="true"
+    fi
+    if [ "${DASHBUILDER_DATASET_PARTITION^^}" = "TRUE" ]; then
+        dataset_partition="true"
+    fi
+
+    JBOSS_KIE_DASHBUILDER_ARGS="${JBOSS_KIE_DASHBUILDER_ARGS} -Ddashbuilder.components.partition=${component_partition}"
+    JBOSS_KIE_DASHBUILDER_ARGS="${JBOSS_KIE_DASHBUILDER_ARGS} -Ddashbuilder.dataset.partition=${dataset_partition}"
+}
+
+function configure_dashbuilder_file_imports() {
+    local kie_data_import_dir="/opt/kie/imports"
```

Comment:
This is different from what I read here:

https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/master/dashbuilder/image.yaml#L45

In the YAML file I read "'/opt/kie/data/imports" - but it is actually '/opt/kie/imports'

Reply:
applied default to /opt/kie/data/imports;